docs/getting-started/endgame-sensor-FDA-ven.asciidoc

```diff
@@ -0,0 +1,110 @@
+[[endgame-sensor-full-disk-access-ven]]
+= Enable Full Disk Access for the Endgame sensor on macOS Ventura and higher
+
+:frontmatter-description: Manually install and deploy Elastic Endgame on macOS Ventura and higher.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [secure] 
+
+If your solution includes Elastic Endgame, it requires Full Disk Access to protect you from malware and other cybersecurity threats. Full Disk Access permissions is a privacy feature that controls which applications can access your data. This means you need to manually enable Full Disk Access permission for the Endgame sensor to access these protected areas of your Mac.
+
+[discrete]
+[[macos-permissions-ven]]
+== macOS permissions
+
+The behavior of the Endgame sensor differs based on your macOS version. MDM/Jamf users can pre-approve all Full Disk Access without granting permission to the sensor. However, depending on the macOS version and sensor type, non-MDM/Jamf users may be prompted to enable Full Disk Access for required security files.
+
+Here are the following Full Disk Access requirements for specific versions of macOS:
+
+- `10.15`: You cannot install the sensor without allowing it to load a <<kernel-ext-approval-endgame, kernel extension>>. During installation, you'll be prompted to go to System Preferences and approve it. Upon approval, installation proceeds.
```

Comment:
Confirmed with @ricardoungureanu that we don't need this information for Ventura. Can remove.

```suggestion
```